test/test_helper.rb

```diff
@@ -7,60 +7,48 @@
 db_sets = {
   "sqlite3" => {
     adapter: 'sqlite3',
-    name: ':memory:'
+    database: ':memory:'
   },
   "mysql56" => {
     host: 'mysql56',
     adapter: 'mysql2',
     username: 'root',
     password: 'password',
-    name: 'test'
+    database: 'test',
+    encoding: 'utf8mb4',
+    charset: 'utf8mb4'
   },
   "mysql57" => {
     host: 'mysql57',
     adapter: 'mysql2',
     username: 'root',
     password: 'password',
-    name: 'test'
+    database: 'test'
   },
   "mysql80" => {
     host: 'mysql80',
     adapter: 'mysql2',
     username: 'root',
     password: 'password',
-    name: 'test'
+    database: 'test'
   },
   "pg12" => {
     host: 'pg12',
     adapter: 'postgresql',
     username: 'postgres',
-    name: 'test'
+    database: 'test'
   },
 }
 db = db_sets[ENV["DB"]] || db_sets['sqlite3']
 
-if db[:adapter] != 'sqlite3'
-  ActiveRecord::Base.establish_connection(
-    host: db[:host],
-    adapter: db[:adapter],
-    username: db[:username],
-    password: db[:password],
-  #  database: db[:name]
-  )
+unless db[:adapter] == 'sqlite3'
+  ActiveRecord::Base.establish_connection(db.slice(:host, :username, :password, :adapter))
```

Comment:
ActiveSupport adds some syntactic sugar to Hash:
```suggestion
  ActiveRecord::Base.establish_connection(db.except(:database))
```

Reply:
moved to #5